keps/168-pending-workloads-visibility/README.md

```diff
@@ -0,0 +1,349 @@
+# KEP-168: Pending workloads visibility
+
+<!--
+This is the title of your KEP. Keep it short, simple, and descriptive. A good
+title can help communicate what the KEP is and should be considered as part of
+any review.
+-->
+
+<!--
+A table of contents is helpful for quickly jumping to sections of a KEP and for
+highlighting any additional information provided beyond the standard KEP
+template.
+
+Ensure the TOC is wrapped with
+  <code>&lt;!-- toc --&rt;&lt;!-- /toc --&rt;</code>
+tags, and then generate with `hack/update-toc.sh`.
+-->
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+  - [Notes/Constraints/Caveats (Optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+    - [Time and memory consuming computation of top pending workloads](#time-and-memory-consuming-computation-of-top-pending-workloads)
+    - [Too large local queue object](#too-large-local-queue-object)
+    - [Increased number of localqueue updates](#increased-number-of-localqueue-updates)
+- [Design Details](#design-details)
+  - [Local Queue API](#local-queue-api)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+The enhancement extends the API of LocalQueue and ClusterQueue to expose the
+information about the order of their pending workloads.
+
+## Motivation
+
+Currently, there is no visibility to users of the contests of the queue by a
+user.
+.
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+### Goals
+
+- expose the order of workloads in the LocalQueue and ClusterQueue
```

Comment:
```suggestion
- expose the order of workloads in the LocalQueue
```

Reply:
Exposing the clusterqueue will allow the admin to have full visibility. It was suggested that the clusterqueue should be the higher priority work (https://github.com/kubernetes-sigs/kueue/pull/991#discussion_r1268779937). Let me leave both for now.